security/name-that-hash/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                name-that-hash
+version             1.0.3
+revision            0
+categories-prepend  security
+platforms           darwin
+license             GPL-3
+supported_archs     noarch
+
+python.default_version \
+                    39
+
+maintainers         {@harens gmail.com:harensdeveloper} \
+                    openmaintainer
+
+description         The Modern Hash Identification System
+long_description    Don't know what type of hash it is? Name \
+                    That Hash will name that hash type! Identify MD5, \
+                    SHA256 and 3000+ other hashes.
+
+homepage            https://nth.skerritt.blog
+
+checksums           rmd160  ed1a9696078752c0f1e58fb0defc016c66a61d1e \
+                    sha256  a1350729fb866b9c3ebc0c4687369c8bbca9253357beb4f7405b6a74580db1a5 \
+                    size    24523
+
+depends_lib-append \
+    port:py${python.version}-setuptools
+
+depends_run-append \
+    port:py${python.version}-click \
+    port:py${python.version}-rich \
+    port:py${python.version}-loguru \
```

Comment:
Two issues here. Please sort these alphabetically. Also please remove the trailing slash.

Reply:
Thanks @mf2k. Changes made.